SB2018/2018-01-12.md

```diff
@@ -0,0 +1,34 @@
+---
+to: All teams
+subject: SourceBots Mentoring and Tech Days
+---
+
+Hi all,
+
+We hope you're progressing well on your robot!
+Some important things to announce so keep reading:
+
+## Mentoring
+
+We're going to try and provide mentoring support for your team!
```

Comment:
Are we able to support everyone, or is there a variant of this email for the teams who we can't mentor?